Server/src/GameEngine/Application/Application.hpp

```diff
@@ -55,9 +56,9 @@ class Application {
          */
         void _packetHandler(Network::UDPPacket packet, const asio::ip::udp::endpoint& endpoint);
 
-        std::shared_ptr<Network::Server>    _server;    // Network class for server.
-
-        ECS::Registry                       _registry;  // registry class for ECS management.
+        std::shared_ptr<std::vector<ECS::Registry>>             _registries;        // vector of registries class for ECS management.
+        std::shared_ptr<SceneManager::ServerSceneManager>       _sceneManager;      // load and handle scene in the ECS.
+        std::shared_ptr<Network::Server>                        _server;            // Network class for server.
```

Comment:
Why is this a shared pointer ?

Reply:
Same as the SceneManager.

Reply:
Same for the Network, since it isn't an interface, has a constructor and don't need to be shared with another class (maybe I'm wrong with the last point I'll let you tell me if I am), it doesn't need to be a shared pointer 👍

Reply:
That's mb